src/router/routes.ts

```diff
@@ -30,6 +30,17 @@ const routes: RouteRecordRaw[] = [
       }
     ]
   },
+  {
+    path: '/mobile-search',
+    component: () => import('layouts/Auth.vue'),
+    children: [
+      {
+        path: '',
+        name: 'mobile-search',
+        component: () => import('components/core/MobileBar.vue')
+      }
+    ]
+  },
```

Comment:
Please remove this route. We will have to find a different solution for expanding the searchbar over the header